Controllers/ComputerController.cs

```diff
@@ -0,0 +1,155 @@
+﻿//Author: Jordan W. Williams
+//Purpose: Allow a user to communicate with the Bangazon database to GET PUT POST and DELETE entries.
+//Methods: GET PUT POST DELETE
```

Comment:
Every method must be documentation with purpose and argument list - which itself must contain a short purpose for each argument.

Reply:
per employee handbook